lib/galaxy/managers/markdown_util.py

```diff
@@ -822,4 +822,5 @@ def _validate(*args, **kwds):
     "ready_galaxy_markdown_for_export",
     "ready_galaxy_markdown_for_import",
     "resolve_invocation_markdown",
+    "validate_galaxy_markdown",
```

Comment:
I will fix this.

Reply:
I mean - I "fixed" this - but it was intentional and I am not happy about it. I didn't want there to be two sources of markdown utilities but that parser file needs to be standalone so it can be reused in gxformat2. The re-export made perfect sense - that detail shouldn't be something consumers within Galaxy care about.